editor/package.json

```diff
@@ -54,7 +54,8 @@
     "dev-fast": "APP_ENV=development REACT_APP_ENVIRONMENT_CONFIG=development REACT_APP_AUTH0_CLIENT_ID=$AUTH0_CLIENT_ID REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_COMMIT_HASH=`git rev-parse HEAD` vite --force",
     "vite-build": "node --max_old_space_size=16384 ./node_modules/vite/bin/vite build -- -DEBUG",
     "watch-tsc": "tsc --watch --noEmit && NODE_OPTIONS=\"--max_old_space_size=4096 $NODE_OPENSSL_OPTION\"",
-    "unused-exports": "ts-unused-exports ./tsconfig.json --ignoreLocallyUsed"
+    "unused-exports": "ts-unused-exports ./tsconfig.json --ignoreLocallyUsed",
+    "eslint-plugin-local-test": "eslint-plugin-local test"
```

Comment:
by running `pnpm eslint-plugin-local-test` we can test our custom rules (we can provide `valid` and `invalid` cases in the rule metadata)